docs/iris/example_tests/extest_util.py

```diff
@@ -69,7 +68,7 @@ def show_replaced_by_check_graphic(test_case, tol=_DEFAULT_IMAGE_TOLERANCE):
     """
     def replacement_show():
         # form a closure on test_case and tolerance
-        test_case.check_graphic(tol=tol)
+        test_case.check_graphic()
```

Comment:
In the hope of refraining from a culture of tweaking graphical testing tolerances, I've closed the door to passing a `tol` through to `check_graphic`.